resources/authgear/templates/en/web/authflowv2/__otp_input.html

```diff
@@ -88,5 +81,16 @@
       </button>
     </form>
   {{ end }}
+
+  <button
+    form="{{ $.FormName }}"
+    class="primary-btn w-full"
+    type="submit"
+    name="{{ or $.SubmitButtonName "x_action" }}"
+    value="{{ or $.SubmitButtonValue "submit" }}"
+    data-otp-input-target="submit"
+    data-authgear-event="{{ $.SubmitEvent }}"
+  > Next </button>
```

Comment:
Label need to be localised

Reply:
{{ template "v2-button-label-continue" }}